script.js

```
@@ -32,3 +32,30 @@ function getCurrentUser() {
+async function fetchBinancePrice(pair) {
+  const url = `https://corsproxy.io/https://api.binance.com/api/v3/ticker/price?symbol=${pair}`;
+  const res = await fetch(url);
+  const data = await res.json();
+  return parseFloat(data.price);
+}
+
+async function fetchBybitPrice(pair) {
+  const url = `https://corsproxy.io/https://api.bybit.com/v5/market/tickers?category=spot&symbol=${pair}`;
+  const res = await fetch(url);
+  const data = await res.json();
+  if (data.result && data.result.list && data.result.list.length > 0) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)